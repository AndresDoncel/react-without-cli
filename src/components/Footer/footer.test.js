import React from "react";
import renderer from "react-test-renderer";

import { Footer } from "./index";

it("renders correctly when there are no items", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
