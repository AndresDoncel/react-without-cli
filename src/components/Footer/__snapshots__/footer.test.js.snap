// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="container__footer"
>
  <p>
    netflix
    <span>
      roulette
    </span>
  </p>
</div>
`;
